packages/@aws-cdk/aws-codedeploy/lib/pipeline-action.ts

```diff
@@ -0,0 +1,81 @@
+import actions = require('@aws-cdk/aws-codepipeline-api');
+import cdk = require('@aws-cdk/cdk');
+
+/**
+ * Construction properties of the {@link PipelineDeployAction CodeDeploy deploy CodePipeline Action}.
+ */
+export interface PipelineDeployActionProps extends actions.CommonActionProps {
+    /**
+     * The name of the CodeDeploy application to deploy to.
```

Comment:
So that everyone is super clear on what's going on, can you rename this to `applicationName` and note in the docstrings that this is a temporary API that will disappear until we get L2s?